tests/Makefile

```diff
@@ -50,9 +50,14 @@ clean:
 install: exes
 	@ echo "installing exes in $(DESTBIN)..." && \
 	mkdir -p $(DESTBIN) && \
-	install -D -v -m u=rwx,g=rx,o=rx copybw -t $(DESTBIN) && \
-	install -D -v -m u=rwx,g=rx,o=rx copylat -t $(DESTBIN) && \
-	install -D -v -m u=rwx,g=rx,o=rx apiperf -t $(DESTBIN) && \
-	install -D -v -m u=rwx,g=rx,o=rx sanity -t $(DESTBIN)
+	install -D -v -m u=rwx,g=rx,o=rx gdrcopy_copybw -t $(DESTBIN) && \
+	install -D -v -m u=rwx,g=rx,o=rx gdrcopy_copylat -t $(DESTBIN) && \
+	install -D -v -m u=rwx,g=rx,o=rx gdrcopy_apiperf -t $(DESTBIN) && \
+	install -D -v -m u=rwx,g=rx,o=rx gdrcopy_sanity -t $(DESTBIN)
+	cd $(DESTBIN) && \
+	ln -s gdrcopy_copybw copybw && \
```

Comment:
you might need to check what happens when there are existing symlinks, i.e. are they overwritten correctly?

Reply:
Nice catch! Fixed.

Reply:
do you need 'ln -sf ' to remove possibly existing destination files?